performanceMetrics/src/App/App.js

```diff
@@ -121,14 +122,14 @@ const App = (props) => {
 		let componentMetrics = [], promises = [];
 
 		for (let version of listOfVersions) {
-			promises.push(axios.get('./' + version + '/' + selectedComponent + '.txt'));
+			promises.push(fetch('./' + version + '/' + selectedComponent + '.txt').then(result => result.text()));
 		}
 
 		Promise.allSettled(promises).then((results) => {
-			const successfulResults = results.filter((result) => result.status === 'fulfilled');
+			const successfulResults = results.filter((result) => result.value.includes('EnactVersion'));
```

Comment:
Result object seems to have the same structure as the one in the screenshot below:
{"ReactVersion":"17.0.2","EnactVersion":"4.1.1","SandstoneVersion":"2.1.1","timestamp":1641787551304,"component":"Alert","type":"DCL","actualValue":824.252}
I think maybe you could filter by 'ReactVersion' here as well, to avoid any confusion that there is a specific reason why you filtered differently.

Reply:
Okay, thanks for noticing